raspberry/anemometer/read_and_save_trisonica_mini_data.py

```diff
@@ -0,0 +1,69 @@
+#!/usr/bin/env python3
```

Comment:
I suppose the purpose of this file is to gather wind data and store them in a file. However, we can do this with our implementation in `sensors/anemometer.py`. That said, do we need the Python files in the folder `anemometer`?